### DISCUSSION TITLE
Add support for analog inputs

### DISCUSSION THREAD

**Discussion by kayhe** [zauberzeug/lizard#47] [Ideas]
For our project, we need to be able to read out sensors via the ADCs. Doing this in Lizard would greatly simplify our software architecture.
The raw voltage from the ADC is enough, so implementing a custom module for the sensor alone is likely overkill.

I started working on an implementation already. If you have any pointers or tips, I'm all ears.

**Comment by falkoschindler** [comment]
Thanks for the suggestion, @kayhe! Such a module would be a great addition to Lizard.

@JensOgorek Would you like to have a look into this feature request?

**Reply by JensOgorek** [reply]
Yeah, I will take a look at it.

Edit: Found lots of useful information about it and started to create a module that can handle ADC.

**Comment by JensOgorek** [comment]
I just created a pull request (#50) with a working ADC module for Lizard.
